int32 x
int32 y
---
Point real_coordinate
